src/components/30-organisms/footer/e2e.js

```diff
@@ -6,11 +6,11 @@ test.describe('footer', () => {
     await page.goto(fixtureURL('examples-footer-react--resize-dynamic'));
 
     const footerBoundingBoxDefault = await page.locator('footer').boundingBox();
-    expect(footerBoundingBoxDefault.height).toBe(314.6875);
+    const initialHeight = footerBoundingBoxDefault.height;
 
     await page.locator('#footerTestButton').click();
 
     const footerBoundingBoxAfter = await page.locator('footer').boundingBox();
-    expect(footerBoundingBoxAfter.height).toBe(278.75);
+    expect(footerBoundingBoxAfter.height).toBeLessThan(initialHeight);
```

Comment:
Good! Only thing I asked myself is whether to prescribe a minimum offset for less-than (suppose it is half a pixel less...). Think about it.